Assets/Scripts/GamePreparation/PreparationMain.cs

```diff
@@ -139,6 +138,7 @@ public void MoveCardToCollection(SimpleCard card)
     public void StartGameClicked()
     {
         Debug.Log("startGameClicked");
+        logic.SaveToFile(System.IO.Path.Combine(gameObject.scene.path, "../playerSelection.xml"));
```

Comment:
Generally speaking, instead of passing array from one place to another we want to save card selection on local machine after card pack is created (first scene) and get it from there on the second scene (battleground)

Reply:
Can we decouple it to separate method in order to make it more visible for the future if point above is correct? The thing is (at least how I see that) that we don't want to store it on local machine (in the future) but on some db probably. So once we at that point, we're able to simply identify the place that will (and should) be changed

Reply:
Yes, it will be much easier to work on MainScene with saved array without need of selection cards each time in CardSelection scene. And yeah, that is not going to be part of final solution. I'll move it to separate method 👍